assets/prototype/application/services/toaster/LoadingToast.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Spinner } from '@blueprintjs/core';
+import { LoadingToastProps } from './types';
+
+const LoadingToastNotice: React.FunctionComponent<LoadingToastProps> = ({ message }): JSX.Element => {
```

Comment:
`React.FC` 😄

Reply:
DOH... did that before we discussed the change and forgot to go back and check!

will fix